html-css/week2/Neepun_orderPage/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/styles.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <header class="header">
+    <h1>Payment Header</h1>
+    <p>Plesae Enter Payment Details</p>
+
+  </header>
+  <main class="main">
+    <section class="paymentSection">
+      <!-- Order Summary Container -->
+      <div class="summaryContainer">
+        <div class="orderSummary">
+          <h2>Order Summary</h>
+            <div class="amountContainer">
+              <div class="totalContainer">
+                <p>Total</p>
+                <p>$333 / month</p>
+              </div>
+              <hr />
+              <div class="bigTotalContainer totalContainer">
+                <p>Total</p>
+                <p>$100</p>
+              </div>
+            </div>
+        </div>
+      </div>
+
+      <!-- Payment Method Container -->
+      <div class="paymentContainer">
+        <h2>Payment Method</h2>
+        <div class="visaContainer">
+          <div class="visaCards paymentWrapper" tabindex="0">
+            <img src="./Images/cards.png" alt="visa cards" />
+            <p>Credit/Debit</p>
+          </div>
```

Comment:
You've used `tabindex` in the effort to make this keyboard accessible. The best way to handle it would have been to make the `<div>` a `<button>`, and remove the `<p>` tag, as `<button>` anyways assumes text inside.
```suggestion
          <button type="button" class="visaCards paymentWrapper">
            <img src="./Images/cards.png" alt="visa cards" />
            Credit/Debit
          </button>
```

Reply:
Another approach would be to use a bit of ugly CSS selectors with `<input type="radio>` - https://codepen.io/menian/pen/raBZZmZ